frontend/src/app/AppContext.ts

```diff
@@ -15,9 +14,14 @@ const defaultAppContext: AppContextProps = {
   setIsNavOpen: () => undefined,
   onNavToggle: () => undefined,
   buildStatuses: [],
-  dashboardConfig: blankDashboardCR,
+  /* DO NOT DO THIS!!!!!!!!!!! :( */
+  // dashboardConfig will never be null during runtime
+  // tests will just mock this
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  dashboardConfig: null,
```

Comment:
Can you try one other things -- talked to another dev and he said maybe `as DashboardConfig` might work. Just trying to set the `null` around TS is all we need to do.